website/integrations/services/home-assistant/index.md

```diff
@@ -16,6 +16,9 @@ sidebar_label: Home Assistant
 :::caution
 You might run into CSRF errors, this is caused by a technology Home-assistant uses and not authentik, see [this GitHub issue](https://github.com/goauthentik/authentik/issues/884#issuecomment-851542477).
 :::
+:::caution
+Only prefixes starting with `/auth` need to be proxied (excluding prefixes starting with `/auth/token`), see [this GitHub issue](https://github.com/BeryJu/hass-auth-header/issues/212). This can be configured in the reverse proxy (e.g. nginx, Traefik) or in Authentik Provider's **Unauthorized Paths**.
```

Comment:
```suggestion
Only prefixes starting with `/auth` need to be proxied (excluding prefixes starting with `/auth/token`), see [this GitHub issue](https://github.com/BeryJu/hass-auth-header/issues/212). This can be configured in the reverse proxy (e.g. nginx, Traefik) or in authentik Provider's **Unauthorized Paths**.
```

Reply:
Thanks so much @sbogomolov just the tiny not of not capitalizing authentik, then we are good. Thank you for your patience with this; I learned a lot, for sure. ;-)

Reply:
Could you squash commits when merging?